jf_agent/validation.py

```diff
@@ -93,6 +99,29 @@ def validate_jira(config, creds):
                 return False
 
 
+def validate_num_repos(git_configs, creds):
+    for git_config in git_configs:
+        for cred_slug in creds.git_instance_to_creds:
+            for project in git_config.git_include_projects:
+                client = None
+                token = creds.git_instance_to_creds.get(cred_slug).get('github_token')
+                try:
+                    client = GithubGqlClient(base_url=git_config.git_url, token=token)
+                    repo_count = client.get_repos_count(login=project)
+                    client.session.close()
+                    msg = f"credentials preface: {cred_slug} " \
+                          f"identified {repo_count} repos in instance {git_config.git_instance_slug}"
+                    agent_logging.log_and_print(logger, logging.WARNING, msg=msg)
```

Comment:
Should this be a `logging.INFO` level?

Reply:
yep, swapped here: https://github.com/Jellyfish-AI/jf_agent/pull/248/commits/2bfdbd50c418a9336b588881aebbe9e79c13ce57#diff-53a011a4f83bc5fb36e2b2215e792e2c0527cda7df39d1cf09c137eb568a7cb0R114